scripts/omni_apply.py

```diff
@@ -11,18 +11,30 @@
 o.add_option('--xtalk',dest='xtalk',default=False,action='store_true',
             help='Toggle: apply xtalk solutions to data. Default=False')
 o.add_option('--omnipath',dest='omnipath',default='%s.fits',type='string',
-            help='Format string (e.g. "path/%s.npz", where you actually type the "%s") which converts the input file name to the omnical npz path/file.')
+            help='Format string (e.g. "path/%s.fits", where you actually type the "%s") which converts the input file name to the omnical npz path/file.')
 o.add_option('--firstcal', action='store_true', 
             help='Applying firstcal solutions.')
 opts,args = o.parse_args(sys.argv[1:])
 args = np.sort(args)
+pols = opts.pol.split(',')
 
+def file2pol(filename): return filename.split('.')[3] #XXX assumes file naming format
```

Comment:
Ugh, the way that we are hardcoding the file naming format is very gross, but it does seem like the only method we have right now. We should embed metadata in the UV files so that we can actually do this correctly.